src/screens/users/list/Users.tsx

```diff
@@ -134,46 +145,49 @@ export default class Users extends BaseAutoRefreshScreen<Props, State> {
     await this.refresh();
   };
 
-  public render = () => {
+  public render() {
     const style = computeStyleSheet();
     const { users, count, skip, limit, refreshing, loading } = this.state;
-    const { navigation } = this.props;
+    const { navigation, isModal, selectionMode, onUserSelected, initiallySelectedUsers } = this.props;
     return (
       <Container style={style.container}>
-        <HeaderComponent
-          title={this.title ?? i18n.t('sidebar.users')}
-          subTitle={count > 0 ? `${I18nManager.formatNumber(count)} ${I18n.t('users.users')}` : null}
-          navigation={this.props.navigation}
-          leftAction={this.onBack}
-          leftActionIcon={'navigate-before'}
-          rightAction={() => {
-            navigation.dispatch(DrawerActions.openDrawer());
-            return true;
-          }}
-          rightActionIcon={'menu'}
-        />
+        {!isModal && (
```

Comment:
I would expect the user's header to be displayed here with the modal title (sub-title is number of users selected/total: 2/180) in the place of the modal header.